client/src/logic/fbdb/fbdb-reducer.ts

```diff
@@ -0,0 +1,27 @@
+import {AnyAction} from 'redux'
+
+type FbState = {
+	uuid: string
```

Comment:
Is there a reason we now ahve the uuid/stats state in both session & fbdb reducer?